backend/utils/__init__.py

```diff
@@ -143,3 +145,17 @@ def get_version() -> str | None:
             return None
         branch = [a for a in output.split("\n") if a.find("*") >= 0][0]
         return branch[branch.find("*") + 2 :]
+    
+
+def check_new_version() -> str | None:
+    response = requests.get("https://api.github.com/repos/zurdi15/romm/releases/latest")
```

Comment:
did you test how much this request slows down first load? like how long this query takes to resolve

Reply:
This is extremelly fast, and if you reach the api request limit it's instant. In any case I'm going to add a half second ``timeout`` just in case it get's stuck for some reason